base/linalg/blas.jl

```diff
@@ -496,18 +549,18 @@ gemv(tA, A, x)
 """
     gbmv!(trans, m, kl, ku, alpha, A, x, beta, y)
 
-Update vector `y` as `alpha*A*x + beta*y` or `alpha*A'*x + beta*y` according to `trans` ('N'
-or 'T'). The matrix `A` is a general band matrix of dimension `m` by `size(A,2)` with `kl`
-sub-diagonals and `ku` super-diagonals. Returns the updated `y`.
+Update vector `y` as `alpha*A*x + beta*y` or `alpha*A'*x + beta*y` according to `trans` (`'N'` (meaning not transposed)
```

Comment:
Really what I think we should do is just put a big damn table of what all these chars mean at the top of the `BLAS` section